examples/stats/exporter/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@opencensus/examples",
+  "version": "0.0.8",
+  "description": "OpenCensus is a toolkit for collecting application performance and behavior data.",
+  "repository": "census-instrumentation/opencensus-node",
+  "scripts": {},
+  "keywords": [
+    "opencensus",
+    "nodejs",
+    "tracing",
+    "profiling"
+  ],
+  "author": "Google Inc.",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.0"
+  },
+  "files": [],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "jshint": "^2.9.7"
```

Comment:
Can we use ```gts``` here? This is what we use everywhere..

Reply:
Fixed: 42f43e11c81d025057cfc57c5217aec9095d2aff